crawler/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import logging.config
+import os
+from http import HTTPStatus
+
+from flask import Flask
+from flask_apscheduler import APScheduler
+
+scheduler = APScheduler()
+
+
+def create_app() -> Flask:
+    app = Flask(__name__)
+    app.config.from_object(os.environ["SETTINGS_MODULE"])
+
+    # setup logging
+    logging.config.dictConfig(app.config["LOGGING"])
+
+    if app.config.get("SCHEDULER_RUN", False):
+        scheduler.init_app(app)
+        scheduler.start()
+
+    @app.get("/health")
```

Comment:
For this healthcheck, I think we should check that there is at least 1 job scheduled. I believe the API for APScheduler makes that available.